simulator/simulator_test.go

```diff
@@ -84,3 +84,36 @@ func TestGetRandom(t *testing.T) {
 	}
 	t.Log(result)
 }
+
+// The default EK modulus returned by the simulator when using a seed of 0.
+func zeroSeedModulus() *big.Int {
+	mod := new(big.Int)
+	mod.SetString("28822369077247917472678792367222000492900687607266016203396010837658035689659299734197520553448174323748099793056802958016858756507381032370846534085226253302163899570149199884764519084423882009496754412303818542106327689835389505182193570218605954083532946205386443901420426483800949411692505321722270168362120233669126618971767634850592961537085580034378654792382721745037008114868292963660065852214336139344092437160292248180730141651685121689744852235653454825312462533593592821458433769751868260970774226182524156494472225018947922698640025954940757779839947653936267356452628876700038270975372202071693935689021", 10)
+	return mod
+}
+
+func TestFixedSeedExpectedModulus(t *testing.T) {
+	s, err := GetWithFixedSeedInsecure(0)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer s.Close()
+
+	modulus := getEKModulus(t, s)
+	if modulus.Cmp(zeroSeedModulus()) != 0 {
+		t.Fatalf("Got %v expected %v", modulus, zeroSeedModulus())
```

Comment:
I think asserts are meant to have error messages that stand alone (though not 100% on this one).

Thoughts on: `getEKModulus() = %v, want %v` ?

Reply:
Good idea, changed